lib/trackler/implementation.rb

```diff
@@ -18,6 +18,11 @@ def initialize(track, problem)
       @problem = problem
     end
 
+    def initialize_copy(original)
+      super
+      @files = original.files.dup
+    end
```

Comment:
This will cause a potentially unnecessary construction of the files hash.
Given the only place this is being used is in x-api to modify the files hash, having it created is not such a bad thing.